ethereum/aave/insert_aave_daily_atoken_balances.sql

```diff
@@ -0,0 +1,136 @@
+CREATE OR REPLACE FUNCTION aave.insert_aave_daily_atoken_balances(start_time timestamptz, end_time timestamptz) RETURNS integer
```

Comment:
```
CREATE FUNCTION
 insert_aave_daily_atoken_balances
-----------------------------------
                                 0
(1 row)

INSERT 0 1
```

Something went wrong here or is this correct?

Reply:
Ah might need to run something below first I guess.

Reply:
Yeah aave."aave_tokens" should be first, then likely [insert_aave_daily_interest_rates](https://github.com/duneanalytics/abstractions/pull/700/files#diff-b6a8eae9914b24dcb8bac4a08754a7a6d947dc34e8d18b82d2ee1be3dff59c8f), then this